builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -201,16 +203,27 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 							Computed: true,
 						},
 
-						// TODO: Imprement ipv6 parameters to be optional
```

Comment:
It's some old comment that I removed from the code, since now we have IPv6 support. :smile: